cluster/test_cluster.go

```diff
@@ -67,7 +67,19 @@ func NewForT(t *testing.T, dv, k, n, seed int, opts ...func(*Definition)) (Lock,
 		}
 
 		feeRecipientAddr := testutil.RandomETHAddress()
-		reg := getSignedRegistration(t, rootSecret, feeRecipientAddr, eth2util.Goerli.Name)
+
+		// get the forkHash to retrieve the network name
+		var def Definition
+		for _, opt := range opts {
+			opt(&def)
+		}
+
+		networkName, _ := eth2util.ForkVersionToNetwork(def.ForkVersion)
```

Comment:
```suggestion
		networkName, err := eth2util.ForkVersionToNetwork(def.ForkVersion)
		require.NoError(t, err)
```

Reply:
The error is ignored on purpose: if `def.ForkVersion` is empty because it wasn't set with an option function, adding `require.NoError(t, err)` will fail the test erroneously.

Reply:
Reworked to make use of the error.